src/main/java/org/launchcode/liftoffrecipetracker/models/data/RecipeRepository.java

```diff
@@ -0,0 +1,10 @@
+package org.launchcode.liftoffrecipetracker.models.data;
+
+
+import org.launchcode.liftoffrecipetracker.models.Recipe;
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface RecipeRepository extends CrudRepository<Recipe, Integer> {
+}
```

Comment:
This Repository may belong up a level in data folder rather than as a data folder in models